src/WarcraftLegacies.Source/Setup/QuestSetup/TrollQuestSetup.cs

```diff
@@ -8,14 +10,19 @@ public static class TrollQuestSetup
     public static void Setup()
     {
       var zandalar = ZandalarSetup.Zandalar;
-
-      zandalar.StartingQuest = zandalar.AddQuest(new QuestZandalar(Regions.ZandalarUnlock));
-      zandalar.AddQuest(new QuestConquerKul());
-      zandalar.AddQuest(new QuestZulfarrak(Regions.Zulfarrak));
-      zandalar.AddQuest(new QuestZulgurub());
-      zandalar.AddQuest(new QuestGundrak());
-      zandalar.AddQuest(new QuestJinthaAlor());
-      zandalar.AddQuest(new QuestHakkar());
+      if (zandalar != null)
+      {
+        var questZulFarrak = new QuestZulfarrak(Regions.Zulfarrak);
+        var questZandalar = new QuestZandalar(Regions.ZandalarUnlock);
+        zandalar.StartingQuest = zandalar.AddQuest(questZandalar);
+      
+        zandalar.AddQuest(questZulFarrak);
+        zandalar.AddQuest(new QuestConquerKul(Regions.Zulfarrak, questZulFarrak, new List<QuestData> {questZandalar}));
```

Comment:
You're only passing in one quest for the last parameter. Maybe just change the signature to `QuestData` instead of a collection?

Reply:
Ah yea my bad. I set that up cause I had to potentially add QuestGoldenFleet. Will adjust